arrows/mvg/applets/tests/CMakeLists.txt

```
@@ -0,0 +1,4 @@
+include(kwiver-test-setup)
+
+add_test(NAME mvg:applets:track_features COMMAND kwiver track-features --config "${kwiver_test_data_directory}/track_features_testing.conf" "${kwiver_test_data_directory}/aphill_240_1fps_crf32.mp4")
+add_test(NAME mvg:applets:init_cameras_landmarks COMMAND kwiver init-cameras-landmarks --config "${kwiver_test_data_directory}/init_cameras_landmarks_testing.conf" --video "${kwiver_test_data_directory}/aphill_240_1fps_crf32.mp4" --tracks "${kwiver_test_data_directory}/tracks.txt")
```

Use new lines to break long lines and make this more readable.
```suggestion
add_test(NAME mvg:applets:track_features
  COMMAND kwiver track-features
            --config "${kwiver_test_data_directory}/track_features_testing.conf"
            "${kwiver_test_data_directory}/aphill_240_1fps_crf32.mp4")
add_test(NAME mvg:applets:init_cameras_landmarks
  COMMAND kwiver init-cameras-landmarks
            --config "${kwiver_test_data_directory}/init_cameras_landmarks_testing.conf"
            --video "${kwiver_test_data_directory}/aphill_240_1fps_crf32.mp4
            --tracks "${kwiver_test_data_directory}/tracks.txt")
```